enigma-swing/src/main/java/cuchaz/enigma/gui/EditableType.java

```diff
@@ -14,8 +14,7 @@ public enum EditableType {
 	FIELD,
 	PARAMETER,
 	LOCAL_VARIABLE,
-	JAVADOC,
-	;
+	JAVADOC;
```

Comment:
Personal opinion: the previous style is nicer for future enum expansion

Reply:
I share your opinion, but I took this over disabling I think two inspections that errored here